src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -83,6 +83,7 @@ export type HostConfig<T, P, I, TI, PI, C, CX, PL> = {
 
   shouldSetTextContent(props : P) : boolean,
   resetTextContent(instance : I) : void,
+  areChildrenOffscreen(props : P) : boolean,
```

Comment:
Regarding the naming, we pass both pending and memoized props to this method. `areChildrenOffscreenForGivenProps` is kind of a mouthful 😆